.github/workflows/lint_build_unit_test.yml

```diff
@@ -13,6 +13,10 @@ jobs:
       uses: actions/checkout@v3
       with:
         submodules: 'recursive'
+        # Fetch depth must be greater than the number of commits included in the push in order to
+        # compare against commit prior to merge. 10 is chosen as a reasonable default for the upper
+        # bound of commits in a single PR.
+        fetch-depth: 10
```

Comment:
Let's make this 15. We do have some recent PRs with more than 10 commits, e.g. (https://github.com/golioth/golioth-firmware-sdk/pull/147)

Reply:
Sounds good!